Matrix<->SMS bot documentation

olmari opened the pull request:
Simple Matrix<->SMS bot and accompanied APK how-to use documentation

Signed-off-by: Sami Olmari <sami@olmari.fi>

olmari committed:
Matrix<->SMS bot documentation

Simple Matrix<->SMS bot and accompanied APK how-to use documentation

Signed-off-by: Sami Olmari <sami@olmari.fi>

rambo reviewed (approved):
Has no top-level heading at first line and setup is top-level heading while it probably should be second level.

But that's just picking nits.

olmari committed:
Refined headers based on feedback

Signed-off-by: Sami Olmari <sami@olmari.fi>

rambo reviewed (approved):
LGTM

olmari committed:
Add general matrix info

Add general info about matrix usage and settings relating especially to new users and hacklab enviroment.

Signed-off-by: Sami Olmari <sami@olmari.fi>

olmari committed:
Fix linebreaks in contents

Signed-off-by: Sami Olmari <sami@olmari.fi>

olmari committed:
Remove extraneous backslash from link.

Signed-off-by: Sami Olmari <sami@olmari.fi>

olmari committed:
Add links to diffirent bridge instructions

Signed-off-by: Sami Olmari <sami@olmari.fi>

olmari committed:
Add instructions for Telegram and Whatsapp -bridges

olmari committed:
Add links to bridge instructions

olmari committed:
Document bridge access using various homeservers

Hacklab.fi homeserver users has automatic access to use our bridges. Hacklab members using other homeservers needs to be added manually to bridges.

Signed-off-by: Sami Olmari <sami@olmari.fi>
